libs/wingsdk/test/target-sim/api.test.ts

```diff
@@ -104,6 +104,62 @@ test("api with one GET route with request params", async () => {
   expect(app.snapshot()).toMatchSnapshot();
 });
 
+test("api with 'name' parameter", async () => {
```

Comment:
Can we also please add a wing sdk test for this?

Reply:
Sure.

Reply:
@eladb 
I have added tests in
examples/tests/valid/api_valid_path.w
examples/tests/valid/api_path_vars.w